content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -13,7 +13,8 @@ The profiles provide customization of the Istio control plane and of the sidecar
 You can start with one of Istio’s built-in configuration profiles and then further customize the configuration for
 your specific needs. The following built-in configuration profiles are currently available:
 
-1. **default**: enables components according to the default [Installation Options](/docs/reference/config/installation-options/)
+1. **default**: enables components according to the default settings of the
+    [`IstioControlPlane` API](/docs/reference/config/istio.operator.v1alpha12.pb/)
```

Comment:
That link points to the schema - to see the default values, either look at istio/operator/data/profiles/default.yaml 
or use istioctl profile dump <profile-name>